operations/mimir-mixin/dashboards/tenants.libsonnet

```
@@ -574,7 +574,7 @@ local filename = 'mimir-tenants.json';
         $.queryPanel(
           [
             'sum(cortex_query_scheduler_queue_length{%(job)s, user="$user"})'
-            % { job: $.jobMatcher($._config.job_names.ruler_query_frontend) },
+            % { job: $.jobMatcher('%s|%s' % [$._config.job_names.ruler_query_frontend, $._config.job_names.ruler_query_scheduler]) },
```

There's no way the `cortex_query_scheduler_queue_length` metric can be exposed by the query-frontend, so let's remove it.

```suggestion
            % { job: $.jobMatcher('%s' % [$._config.job_names.ruler_query_scheduler]) },
```